### PR TITLE
NCS v2.4.4 release

### PR DESCRIPTION
**Pull request by gmarull**

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -118,7 +118,7 @@ manifest:
           compare-by-default: false
     - name: mcuboot
       repo-path: sdk-mcuboot
-      revision: v1.10.0-ncs1-1
+      revision: v1.10.0-ncs1-2
```
Comment (mariusz-nordicsemi) [inline west.yml:121]: no update needed

File: west.yml [hunk 2]
```diff
@@ -127,19 +127,19 @@ manifest:
     - name: mbedtls
       path: modules/crypto/mbedtls
       repo-path: sdk-mbedtls
-      revision: v3.3.0-ncs1-1
+      revision: v3.3.0-ncs1-2
```
Comment (mariusz-nordicsemi) [inline west.yml:130]: no update needed

File: west.yml [hunk 3]
```diff
@@ -127,19 +127,19 @@ manifest:
...
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: v2.4.3
+      revision: v2.4.4
     - name: trusted-firmware-m
       repo-path: sdk-trusted-firmware-m
       path: modules/tee/tf-m/trusted-firmware-m
-      revision: v1.7.0-ncs1-1
+      revision: v1.7.0-ncs1-2
```
Comment (mariusz-nordicsemi) [inline west.yml:138]: no update needed

**Review by mariusz-nordicsemi — CHANGES_REQUESTED** [review]

**Comment by gmarull** [comment]
> tools version needs update sample where version is stated needs to be updated

need more details :-)

**Comment by mariusz-nordicsemi** [comment]
> > tools version needs update sample where version is stated needs to be updated
> 
> need more details :-)

You could use updated from 2.7.0 for reference:
https://github.com/nrfconnect/sdk-nrf/commit/bba7549d0c8cb9db470a7e1595a37ba0507cfe80

**Comment by gmarull** [comment]
> > > tools version needs update sample where version is stated needs to be updated
> > 
> > 
> > need more details :-)
> 
> You could use updates from 2.7.0 for reference: [bba7549](https://github.com/nrfconnect/sdk-nrf/commit/bba7549d0c8cb9db470a7e1595a37ba0507cfe80)

such files do not exist in 2.4.x

**Comment by mariusz-nordicsemi** [comment]
> > > > tools version needs update sample where version is stated needs to be updated
> > > 
> > > 
> > > need more details :-)
> > 
> > 
> > You could use updates from 2.7.0 for reference: [bba7549](https://github.com/nrfconnect/sdk-nrf/commit/bba7549d0c8cb9db470a7e1595a37ba0507cfe80)
> 
> such files do not exist in 2.4.x

I've just realized that. And I confirmed with Matter team.

**Review by mariusz-nordicsemi — APPROVED** [review]

**Review by bjornspock — APPROVED** [review]

**Review by shanthanordic — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Review by martintv — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Review by shanthanordic — APPROVED** [review]
Found the root cause of failures, Private repos tags were not pointing to correct #  after correction

**Pull request merged**